### PR TITLE
Drop unwanted images to save 13 MB

### PR DESCRIPTION
**Pull request by lzap**
Plus there is additional patch in livecd-tools to save additional 33 MB!

https://github.com/rhinstaller/livecd-tools/pull/18

Our BMI is back to normal now ;-)

### CODE REVIEW

File: 25-minimize.ks [hunk 1]
```diff
@@ -64,14 +62,20 @@ echo fdi > /etc/hostname
 echo " * locking root account"
 passwd -l root
 
+echo " * store list of packages sorted by size"
+rpm -qa --queryformat '%{SIZE} %{NAME}\n' | sort -n -r > /usr/PACKAGES-LIST
+
 echo " * cleaning up yum cache and removing rpm database"
-rpm -qa > /usr/packages_list
```
Comment (stbenjam) [inline 25-minimize.ks]: Why remove this? It's probably useful for troubleshooting isn't it?

File: 25-minimize.ks [hunk 2]
```diff
@@ -64,14 +62,20 @@ echo fdi > /etc/hostname
...
 # fix the vim syntax markup */
+
+# no more python loading after this step
+echo " * removing python precompiled *.pyc files"
+find /usr/lib64/python*/ /usr/lib/python*/ -name *py[co] -print0 | xargs -0 rm -f
 %end
 
 %post --nochroot
-echo " * disquieting the microkernel boot process"
+echo " * disquieting the boot process"
```
Comment (stbenjam) [inline 25-minimize.ks]: Lol, so many bytes saved here :-) :+1:

File: 25-minimize.ks (hunk 1 above)
Comment (stbenjam) [inline 25-minimize.ks]: Disregard, I see above.

**Comment by stbenjam** [comment]
Merged as 9dfdb79df6bb507bf32c2aada55c3d31686df330, thanks @lzap !

File: 25-minimize.ks (hunk 2 above)
Comment (lzap) [inline 25-minimize.ks]: Every bit counts. Actually, the backgrounds from line 42 were big deal in this patch.

File: 25-minimize.ks (hunk 1 above)
Comment (lzap) [inline 25-minimize.ks]: Actually, the name was referenced from `discovery-debug` therefore I fixed this and pushed.

**Comment by lzap** [comment]
Oh let me fix the `discovery-debug` in a different patch.

**Pull request closed without merging**